charts/common/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: common
 description: Function library for k8s-at-home charts
 type: library
-version: 2.2.1
+version: 2.2.2
```

Since this is added functionality and not so much a bugfix, a 2.3.0 version would have been the way to go. (https://semver.org/)
```suggestion
version: 2.3.0
```